frontend/app/(footer)/register/page.tsx

```diff
@@ -192,13 +192,13 @@ export default function Register() {
           <div>
             <button
               type="submit"
-              className="flex w-full justify-center rounded-md bg-black px-3 py-1.5 text-sm font-semibold leading-6 text-white shadow-sm hover:bg-white hover:text-black focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 border-black border-2"
+              className="flex w-full justify-center rounded-md px-3 py-2 font-semibold leading-6 text-white shadow-sm hover:bg-white hover:text-highlight-yellow focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 border-highlight-yellow bg-highlight-yellow border-2 transition-colors duration-300 transform"
```

Comment:
i think we eventually want to refactor out some of the common tailwind into a tailwind commons file.. so like this animation of `border-highlight-yellow bg-highlight-yellow border-2 transition-colors duration-300 transform` animation will be very common amongst all of our clickables.. possibly 

anyways, thats out of scope for now, just thought ill pop my thought down